src/Akka.Persistence.SqlServer/Akka.Persistence.SqlServer.csproj

```diff
@@ -1,109 +1,44 @@
-    </Reference>
-    <Reference Include="Google.ProtocolBuffers, Version=2.4.1.555, Culture=neutral, PublicKeyToken=55f7125234beb589, processorArchitecture=MSIL">
-      <HintPath>..\packages\Google.ProtocolBuffers.2.4.1.555\lib\net40\Google.ProtocolBuffers.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Google.ProtocolBuffers.Serialization, Version=2.4.1.555, Culture=neutral, PublicKeyToken=55f7125234beb589, processorArchitecture=MSIL">
-      <HintPath>..\packages\Google.ProtocolBuffers.2.4.1.555\lib\net40\Google.ProtocolBuffers.Serialization.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Collections.Immutable, Version=1.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\packages\System.Collections.Immutable.1.3.1\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <AssemblyTitle>Akka.Persistence.SqlServer</AssemblyTitle>
+      <Description>Akka Persistence journal and snapshot store backed by SQL Server database.</Description>
+      <TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
+      <PackageTags>akka;actors;actor model;Akka;concurrency;SQL;SQL Server</PackageTags>
+      <Copyright>Copyright © 2013-2017 Akka.NET Team</Copyright>
+      <Authors>Akka.NET Team</Authors>
+      <VersionPrefix>1.3.0</VersionPrefix>
+      <PackageIconUrl>http://getakka.net/images/akkalogo.png</PackageIconUrl>
+      <PackageProjectUrl>https://github.com/akkadotnet/Akka.Persistence.SqlServer</PackageProjectUrl>
+      <PackageLicenseUrl>https://github.com/akkadotnet/Akka.Persistence.SqlServer/blob/master/LICENSE</PackageLicenseUrl>
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.6' ">1.6.1</NetStandardImplicitPackageVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Akka" Version="1.3.0" />
+      <PackageReference Include="Akka.Persistence" Version="1.3.0" />
+      <PackageReference Include="Akka.Persistence.Sql.Common" Version="1.3.0" />
+      <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+      <PackageReference Include="protobuf" Version="2.6.1.1" />
```

Comment:
Oversight when looking at previous referenced packages - have removed it, along with Newtonsoft